lib/texutil.js

```diff
@@ -42,11 +42,128 @@ module.exports.latex_function_names = arr2set([
+            min = index;
+            first = subs[i];
+        }
+    }
+    return [min, first];
+}
+
+function parseNewCommand(definition, command, cmds) {
+    definition = definition.substring(command.length);
+    var sections = findBracketSections(definition);
+    if (sections.length === 0) {
+        var commands = definition.match(/\\[a-zA-Z]+/g);
+        cmds[commands[0]] = commands[1];
+    }
+    else if (sections.length === 1) {
+        var cmd = definition.replaceAll(sections[0][0], "").match(/\\[a-zA-Z]+/)[0];
+        if (definition.indexOf(cmd) === 0) {
+            cmds[cmd] = sections[0][0];
+        }
+        else {
+            cmds[sections[0][0]] = cmd;
+        }
+    }
+    else if (sections.length === 2) {
+        if (sections[0][1] !== "[") {
+            cmds[sections[0][0]] = sections[1][0];
+        }
+    }
+}
+
+function bracketBalance(string) {
+    var balance = 0;
+    var open = ["(", "[", "{"];
+    for (var index in open) {
+        balance += string.split(open[index]).length - 1;
+    }
+    var close = [")", "]", "}"];
+    for (index in close) {
+        balance -= string.split(close[index]).length - 1;
+    }
+    return balance;
+}
+
+function parseSty(inp, cmds) {
+    var starts = ["\\newcommand", "\\renewcommand", "\\DeclareRobustCommand", "\\defSpecFun"];
+    inp = inp.split("\n").map(function(line) {return line.match(/([^\\%]|\\.|^)+/g)[0];}).join("\n");
+    inp = inp.replace(/\n+/g, "\n").replace(/^\n|\n^/g, "");
+    var lines = inp.split("\n");
+    var lineno = 0;
+    while (lineno < lines.length) {
+        var line = lines[lineno];
+        var command = findFirst(line, starts, 0);
+        var balance = bracketBalance(line);
+        while (balance !== 0 || (lineno + 1 < lines.length && lines[lineno + 1].charAt(0) !== "\\")) {
+            lineno++;
+            balance += bracketBalance(lines[lineno]);
+            line += lines[lineno];
+        }
+        line = line.replace(/\s/g, "");
+        if (command[0] !== -1) {
+            command = command[1];
+            if (command === "\\newcommand" || command === "\\renewcommand") {
+                parseNewCommand(line, command, cmds);
+            }
+            else if (command === "\\DeclareRobustCommand") {
+                var sections = findBracketSections(line);
+                if (sections[0][0] !== undefined && sections[1][0] !== undefined) {
+                    cmds[sections[0][0]] = sections[1][0];
+                }
+            }
+        }
+        lineno++;
+    }
+}
+
 module.exports.paren = arr2set(["\\sinh"]);
+module.exports.other_literals3 = obj2map({});
 try{ //try catch to make sure program still runs without csv file
     var csv = require('csv-parse/lib/sync');
     var fs = require('fs');
     var path = require('path');
+    module.exports.other_literals3 = {};
+    parseSty(fs.readFileSync(path.join(__dirname, "texvc.sty"), 'utf8'), module.exports.other_literals3);
```

Comment:
Why not make parseSty return the array. i.e
`module.exports.other_literals3 =parseSty (...)`